modules/genomic_browser/README.md

```diff
@@ -4,6 +4,24 @@
 
 To use the methylation tab, the genomic_cpg_annotaion table should be filled with data from the Illumina HumanMethylation450k probe annotations file. This file can be found at ftp://webdata2:webdata2@ussd-ftp.illumina.com/downloads/ProductFiles/HumanMethylation450/HumanMethylation450_15017482_v1-2.csv. We provide a script,  `modules/genomic_browser/tools/HumanMethylation450k_annotations_to_sql` to transforme the csv file from Illumina's FTP to into a mysql transaction file. The output of the tool can be piped to mysql. The process should take between 5 to 10 minutes. 
 
-usage :```
+usage :
+```
 python3 HumanMethylation450k_annotations_to_sql.py <annotation_file> | mysql -u <user> -p <database>
 ```
+
+## Files
+
+
+To enable uploading of large files via the genomic file uploader, update apache configurations to increase size and time limits.  (Sample values suggested below)
+
```

Comment:
Also copy here the "Note: " paragraph from in ajax/*.php file below
Add: It's recommended that this configSetting should probably be soft-linked to /data/ path